BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -45,6 +63,9 @@ public struct AccessibilityElement: Element {
     /// See [Accessibility Activate Documentation](https://developer.apple.com/documentation/objectivec/nsobject/1615165-accessibilityactivate) for further information.
     public var accessibilityActivate: (() -> Bool)? = nil
 
+    /// The array contains one or more `Custom Action`s, defining additional supported actions. Assistive technologies, such as VoiceOver, will display your custom actions to the user at appropriate times.
+    public var customActions: [CustomAction]? = nil
```

Comment:
Let's just keep this as an `Array`, empty would mean there's no actions – right?

Reply:
It's just an extra state that doesn't really mean anything – eg what is the difference between `nil` and `[]`?

Reply:
👍🏻